orbit/cmd/orbit/orbit.go

```diff
@@ -1079,6 +1081,20 @@ func main() {
 		checkerClient.GetServerCapabilities().Copy(orbitClient.GetServerCapabilities())
 		g.Add(capabilitiesChecker.actor())
 
```

Comment:
While we're at it, could we print orbit's version to `log.Info()` too? (It will be useful when debugging customer provided logs.)

Reply:
Added orbit info to logs.